crypto/bio/bss_acpt.c

```diff
@@ -434,7 +434,7 @@ static long acpt_ctrl(BIO *b, int cmd, long num, void *ptr)
                 b->init = 1;
             } else if (num == 1) {
                 OPENSSL_free(data->param_serv);
-                data->param_serv = BUF_strdup(ptr);
+                data->param_serv = OPENSSL_strdup(ptr);
```

Comment:
I have not seen this in the other PR, but also here the error handling is missing.

Reply:
> I have not seen this in the other PR,

`BUF_strdup()` is not any more used in current master.

> but also here the error handling is missing.

Good point - corrected, also in #11986.